src/Rotation.cpp

```diff
@@ -204,29 +195,33 @@ namespace ale {
   }
 
 
-  std::vector<double> Rotation::operator()(
-        const std::vector<double>& vector,
-        const std::vector<double>& av
+  ale::Vec3d Rotation::operator()(const ale::Vec3d &vector) const {
+    std::vector<double> tempVec = {vector.x, vector.y, vector.z};
+    Eigen::Map<Eigen::Vector3d> eigenVector((double *)tempVec.data());
```

Comment:
We don't need to use a Map here anymore. This can just make an Eigen::Vector3d object and then unpack it into our Vec3d object.